{"artifact": "success-stories/rainin/rainin@800.jpg", "exception": "RuntimeError: Could not locate imagemagick.", "traceback": "Traceback (most recent call last):\n  File \"/home/ayla/lektor-seantis-website/env/lib/python3.10/site-packages/lektor/builder.py\", line 900, in update\n    yield ctx\n  File \"/home/ayla/lektor-seantis-website/env/lib/python3.10/site-packages/lektor/builder.py\", line 1151, in build_artifact\n    build_func(artifact)\n  File \"/home/ayla/lektor-seantis-website/env/lib/python3.10/site-packages/lektor/imagetools.py\", line 616, in build_thumbnail_artifact\n    process_image(\n  File \"/home/ayla/lektor-seantis-website/env/lib/python3.10/site-packages/lektor/imagetools.py\", line 512, in process_image\n    im = find_imagemagick(ctx.build_state.config[\"IMAGEMAGICK_EXECUTABLE\"])\n  File \"/home/ayla/lektor-seantis-website/env/lib/python3.10/site-packages/lektor/imagetools.py\", line 452, in find_imagemagick\n    raise RuntimeError(\"Could not locate imagemagick.\")\nRuntimeError: Could not locate imagemagick."}
